src/DIRAC/WorkloadManagementSystem/Agent/PilotLoggingAgent.py

```diff
@@ -0,0 +1,89 @@
+""" :mod: PilotLoggingAgent
+
+    PilotLoggingAgent sends Pilot log files to an SE
+"""
+
+# # imports
+import os, requests
+from DIRAC import S_OK, S_ERROR
+from DIRAC.Core.Base.AgentModule import AgentModule
+from DIRAC.Core.Security.Locations import getHostCertificateAndKeyLocation, getCAsLocation
+from DIRAC.DataManagementSystem.Client.DataManager import DataManager
+
+
+class PilotLoggingAgent(AgentModule):
+    """
+    .. class:: PilotLoggingAgent
+
+    The agent sends completed pilot log files to permanent storage for analysis.
+    """
+
+    def initialize(self):
+        """
+        agent's initalisation. Use this agent's CS information to:
+        Determine what Defaults/Shifter shifter proxy to use.,
+        get the target SE name from the CS.
+        Obtain log file location from Tornado.
+
+        :param self: self reference
+        """
+
+        # get shifter proxy for uploads (VO-specific shifter from the Defaults CS section)
+        self.shifterName = self.am_getOption("ShifterName", "GridPPLogManager")
```

Comment:
The Operations section of the CS is already organized by VO. Do not put more specific info in the agent's configuration.

Reply:
OK, I moved all options to the Operation section. The added benefit is that the agents now put be a multi -VO capable. There are some other changes as well. 

However I have a problem with a shifter proxy. Shifter proxy can only be defined in agent's `initalize()` method, otherwise it is not taken into account, this is the first thing `am_go()` does.

Now , the idea was to discover the VOs which are willing to use remote logging, and then used VO-specific parameters to upload files to an SE by looping over VOs in the `execute()` method.

Unfortunately a requirement to set a shifter proxy during the `initialize()` makes it impossible. Or is there a simple solution for that  and I'm  missing something ?

Reply:
The other question is: why would a VO willingly not want to choose remote logging?

Reply:
Even, if they do, you need a VO-agnostic credential to ship data to an SE. Hence a  fallback to `GridPPLogManager`.

Reply:
What is used in TransformationSystem: https://dirac.readthedocs.io/en/latest/AdministratorGuide/Systems/Transformation/index.html#multi-vo-configuration

Reply:
I used this approach, as suggested by @andresailer . It works fine.